Automattic/jetpack
# required-review: Commit yarn.lock

#### Changes proposed in this Pull Request:

Indirect dependencies wind up in the `dist` files, causing random PRs to
make updates to the mirror repo.

Committing yarn.lock should prevent this, as that way only PRs that
explicitly update yarn.lock should result in `dist` changes.

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure that yarn.lock isn't being pushed to the mirror by examining the output of the build step.
* Post-merge, watch for spurious commits like https://github.com/Automattic/action-required-review/commit/48d66a49258d2670a6327e00ea30ffc363b159be to the mirror repo.
